app/src/main/kotlin/com/camerasecuritysystem/client/SettingsActivity.kt

```diff
@@ -15,50 +16,53 @@ import kotlinx.coroutines.launch
 class SettingsActivity : AppCompatActivity(),
     ConnectDialog.ConnectDialogListener {
 
-    private var binding: ActivitySettingsBinding? = null
+    private lateinit var binding: ActivitySettingsBinding
 
     private var keyStore = KeyStoreHelper("connectToServer")
 
-    lateinit var connectionLiveData: ConnectionLiveData
+    private var isFragmentDurationValid: Boolean = false
 
-    lateinit var connectBtn : Button
+    lateinit var connectionLiveData: ConnectionLiveData
 
     private lateinit var sharedPreferences: SharedPreferences
     var pwdIV: String? = null
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_settings)
+        sharedPreferences =
+            this.getSharedPreferences("com.camerasecuritysystem.client", Context.MODE_PRIVATE)
 
-        supportActionBar?.setDisplayHomeAsUpEnabled(true)
         binding = ActivitySettingsBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
 
-        sharedPreferences =
-            this.getSharedPreferences("com.camerasecuritysystem.client", Context.MODE_PRIVATE)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
 
         pwdIV = sharedPreferences.getString(resources.getString(R.string.pwdIVByte), "")
 
-
-        val settings = findViewById<View>(R.id.textViewConnectionSettings)
-        settings.setOnClickListener {
+        binding.textViewConnectionSettings.setOnClickListener {
             openDialog()
         }
 
-        connectBtn = findViewById<View>(R.id.connectBtn) as Button
-
-        connectBtn.setOnClickListener {
+        binding.saveButton.setOnClickListener {
+            saveSettings()
+        }
 
+        binding.connectBtn.setOnClickListener {
             val context = this.applicationContext
 
             //Try to setup a connection
             GlobalScope.launch {
                 try {
                     val connection = ServerConnection.getInstance()
-                    if (!connection.isConnected()){
+                    if (!connection.isConnected()) {
                         connection.connectIfPossible(context)
 
-                        Log.e("Connection", "Connecting is possible: ${connection.connectIfPossible(context)}")
-                    }else{
+                        Log.e(
+                            "Connection",
+                            "Connecting is possible: ${connection.connectIfPossible(context)}"
+                        )
+                    } else {
```

Comment:
This is also something that will very likely give merge conflicts with my branch and has no functional change here, so if you could revert this, great.

Reply:
Altho I will proceed to revert this, the formatter changes this to the way I committed it anyway. So I don't see much issues to be honest.

Reply:
Done

Reply:
Then the issue is that a) we don't have a formatter or predefined formatting style under VC and b) we don't use the formatter